src/utilities/linkBuilder.js

```diff
@@ -26,10 +28,39 @@ export function txPostLink(xdr) {
       },
     },
   });
+  return hashBuilder(SLUG.EXPLORER, query);
+}
 
+export function xdrViewer(xdr, type) {
+  let query = serializeStore(SLUG.XDRVIEWER, {
+    xdrViewer: {
+      input: xdr,
+      type,
+    },
+  });
+  return hashBuilder(SLUG.XDRVIEWER, query);
+}
+
+// explorerEndpoint is pretty low level. Do not use directly
```

Comment:
This should be improved:

-  A functions code comment should explain what it provides or what role it fulfills.
-  Commanding "do not use directly" is not very helpful, instead, offer an alternative or provide instances in which using it would make sense, to help guide the reader.
-  Low level compared to what? Is there a higher level alternative?